source/simulator/parameters.cc

```diff
@@ -134,7 +134,8 @@ namespace aspect
 
     prm.declare_alias ("Use years instead of seconds",
                        "Use years in output instead of seconds",
-                       true);
+                       (Utilities::MPI::this_mpi_process(mpi_communicator)
+                        == 0) ? true: false ); // trigger warning on rank 0
```

Comment:
Nice idea, but `declare_parameters` currently doesnt have access to the mpi communicator. Maybe add that to the interface of the function?